dll/win32/netapi32/na-debug.h

```diff
@@ -0,0 +1,205 @@
+/*
+ * PROJECT:     ReactOS NETAPI32
+ * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:     Debugging NETAPI32
+ * COPYRIGHT:   Copyright 2025 Katayama Hirofumi MZ <katayama.hirofumi.mz@gmail.com>
+ */
+
```

Comment:
I'm a bit bothered by the fact that now, for each module you want to "un-wine-ify", we'll have to duplicate this wine-debug channel functionality.

Reply:
I introduced `wine2ros`.